internal/standby/standby.go

```diff
@@ -107,15 +112,27 @@ func (s *Service) runDetector(ctx context.Context) {
 func (s *Service) CheckForOutage(currentTime time.Time) {
 	outageThreshold := s.cfg.Standby.OutageThreshold
 	timeSinceLastCmd := currentTime.Sub(s.storageSvc.GetCommandTimestamp())
-	s.logger.Debug("checking", "time since last command", timeSinceLastCmd)
+	s.logger.Debug("checking", "time since last command", timeSinceLastCmd, "current mode", s.getMode(), "currentTime", currentTime)
 	if timeSinceLastCmd > outageThreshold {
 		if s.InStandbyMode() {
 			s.logger.Info("Outage detected", "config threshold", outageThreshold, "time since last command", timeSinceLastCmd)
 			s.setMode(CommandMode)
 		} else {
 			s.logger.Debug("Ongoing outage", "config threshold", outageThreshold, "time since last command", timeSinceLastCmd)
+			err := s.planHandler.TrimPlan(currentTime)
```

Comment:
I am not sure, why do we need to trim the plan?

Reply:
I felt like this would be a good idea to start with, then wasn't so sure it was useful by the time I had done it. My thinking was basically that if we are repeatedly reading from a stored plan, removing expired intervals would make it quicker to get to the current one. If you're doing that on every iteration though you are probably spending more time doing the cleanup than you would be just skipping past the stale intervals.

Reply:
I feel that if it isn't needed, that this feature has come up out of necessity, I would probably drop it. It simplifies what we actively do and we may eventually have a different need that makes this unnecessary altogether

Reply:
yeah, in practice our optimisation plans are not so big that just loading the whole thing into memory and iterating through it becomes a concern

Reply:
great, will remove